### DISCUSSION TITLE
Generate DDL for whole table is missing column comments

### DISCUSSION THREAD

**Discussion by elring** [dbeaver/dbeaver#11984] [Q&A]
#### System information: 
- OS: Linux Mint 20
- DBeaver version: 21.0.2 EA, 21.0.0

#### Connection specification:
- Database: Postgresql 12

#### Describe the problem you're observing:
The DDL function (DDL tab) in table detail for table with columns with comments doesn't create those comments.

#### Steps to reproduce
Use any table, for example 
```
CREATE TABLE public.testtable (
	id serial NOT NULL,
	testchar varchar NULL,
	CONSTRAINT testtable_pkey PRIMARY KEY (id)
);
COMMENT ON COLUMN public.testtable.testchar IS 'this is my comment';
```
Try to get DDL for whole table. Use the tab DDL in the table.
The result SQL is:
```
CREATE TABLE public.testtable (
	id serial NOT NULL,
	testchar varchar NULL,
	CONSTRAINT testtable_pkey PRIMARY KEY (id)
);
```
When I use table list function Generate SQL - DDL, it works:
```
-- public.testtable definition

-- Drop table

-- DROP TABLE public.testtable;

CREATE TABLE public.testtable (
	id serial NOT NULL,
	testchar varchar NULL, -- this is my comment
	CONSTRAINT testtable_pkey PRIMARY KEY (id)
);

-- Column comments

COMMENT ON COLUMN public.testtable.testchar IS 'this is my comment';
```

**Comment by LonwoLonwo** [comment]
hello @elring 
Table and column comments are not default part of table DDL, so we added button "Show comments" in table tab "DDL" and in Generate->DDL

![2021-04-04 17_32_14-DBeaver Enterprise 21 0 2 - reminders](https://user-images.githubusercontent.com/45152336/113512181-1a965b00-956c-11eb-8ac6-e046c11ba368.png)

![2021-04-04 17_33_46-DBeaver Enterprise 21 0 2 - reminders](https://user-images.githubusercontent.com/45152336/113512182-1b2ef180-956c-11eb-8c8e-3391bdfd9d47.png)

Are they working?

**Comment by elring** [comment]
OK, my mistake. It works, I just didn't noticed the settings. I'm sorry :)

**Reply by LonwoLonwo** [reply]
No problem :)

**Comment by evillarroya** [comment]
Hello, I am missing this option in DBeaver 23.2.3 for Macos. Is it moved somewhere else? Database is DB2 for IBM iConnection

**Reply by LonwoLonwo** [reply]
Hello @elring 

It is supported for PostgreSQL in DBeaver. You can create a new ticket for DB2, I think

**Reply by ToMakeSense** [reply]
> Hello, I am missing this option in DBeaver 23.2.3 for Macos. Is it moved somewhere else? Database is DB2 for IBM iConnection

Same here,  vesion: `23.3.1.202312241705`  for Oracle on Windows-10
